index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap"
+      rel="stylesheet"
+    />
+
+    <title>Document</title>
+
+    <style></style>
+  </head>
+  <body>
+    <section class="home">
+      <div class="container">
+        <nav class="first-nav">
+          <h2 class="header-logo"><a href=""> My Logo </a></h2>
+          <div class="hamburger"><img src="./images/union.svg" alt="un" /></div>
+        </nav>
+        <h1 class="heading_second">
+          I'm Silvia<br />
+          Glad to see <br />
+          you!
+        </h1>
+        <div class="first-para">
+          <p>
+            I'm a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don't hestiate to
+            contact me.
+          </p>
+        </div>
+        <p class="second-para">LET'S CONNECT</p>
+        <div class="social_links">
+          <img src=./images/Vector-2.jpg class=logo-icons alt=union> <img
+          src=./images/Linkedin-icon.jpg class=logo-icons alt=union> <img
+          src=./images/Vector-3.jpg class=logo-icons alt=union> <img
+          src=./images/Vector-4.jpg class=logo-icons alt=union> <img
+          src=./images/Vector.jpg class=logo-icons alt=union>
+        </div>
```

Comment:
Last but not least, the list of icons should use a `ul` tag to group them, and `li` per icon. Each icon should be a link (`a` tag) because it must be clickable in order to redirect you to the clicked social media website.

The structure should look more or less like like:
![image](https://user-images.githubusercontent.com/38088818/164534770-756b0e62-de52-4aa9-8ec2-372c6f440880.png)

And of course, you might also need to update your CSS to accommodate the restructuring of your entire page.
Good luck and happy coding🥂

Reply:
ok, thank you so much I will do it and submit it again

Reply:
so, I follow you instruction and I obtain
http://127.0.0.1:5501/index.html
because if I put <ul><a href="a">for every icon</a></ul> the icons allign vertical not horizontal

Reply:
so, what I should do next for approv, let it the same that the repo was or git push from my Visual studio with your recomandations that I follow and obtain different mobile skeleton?

Reply:
Hi @silviatofana,

If your design does not look properly after making the changes, try to restructure your CCS selectors as well so that the way you apply your styles is aligned with the new HTML structure you implemented. Your mobile view should still look the same way it was before you restructured your HTML.

Kindly makes all the necessary changes, and remember to push your changes to this branch and re-submit this PR again for a re-review when you are confident with your changes. 😊 

Good luck and happy coding 🥂

Reply:
ok, I did it and give again submit for another review please
thanks for all yuor advices

Reply:
It's always a pleasure, always here to help 🤗